packages/builders/scripts/vats/upgradeVaults.js

```diff
@@ -7,12 +7,18 @@ export const defaultProposalBuilder = async ({ publishRef, install }) =>
     getManifestCall: [
       'getManifestForUpgradeVaults',
       {
-        vaultsRef: publishRef(
+        VaultFactoryRef: publishRef(
           install(
             '@agoric/inter-protocol/src/vaultFactory/vaultFactory.js',
             '../bundles/bundle-vaultFactory.js',
           ),
         ),
+        contractGovernorRef: publishRef(
+          install(
+            '@agoric/governance/src/contractGovernor.js',
+            '../bundles/bundle-contractGovernor.js',
+          ),
+        ),
```

Comment:
Looks like we're installing the governor from two places. So we'll get 2 installation handles.

Does it matter whether the governor for VaultFactory has an identical installation handle to the one from the auctioneer instance? Does it matter that only one of them can be checked against `agoricNames.installation` to see whether it's the "right" one?

Reply:
We discussed the possibility of only using this mechanism in 1 place and waiting for it in the other. (take care to not just wait for `consume.installation.contractGovernor`, since that's already resolved)

It doesn't seem that either option is catastrophic, so I'll leave it at your discretion.